libr/bin/bfile.c

```diff
@@ -936,13 +930,32 @@ R_API RList *r_bin_file_hash(RBin *bin, ut64 limit) {
 		sha256h->hex = strdup (hash);
 		r_list_push (file_hashes, sha256h);
 	}
+
+	if (o->plugin && o->plugin->hashes) {
+		RList *plugin_hashes = o->plugin->hashes (bf);
+		if (!r_list_empty (plugin_hashes)) {
+			r_list_join (file_hashes, plugin_hashes);
+		}
+		free (plugin_hashes);
+	}
 	// TODO: add here more rows
 
 	free (buf);
 	r_hash_free (ctx);
 	return file_hashes;
 }
 
+R_API RList *r_bin_file_set_hashes(RBin *bin, RList/*<RBinFileHash*/ *new_hashes) {
```

Comment:
please add a comment here, explaining that the returned RList should be freed by the caller and the new one will be owned by RBin.